.github/workflows/platform-docker-build-e2e-image.yml

```diff
@@ -0,0 +1,28 @@
+name: Build E2E Frontend Image
+
+on:
+  schedule:
+    # Update the E2E Firefox testcafe version on the first of every month
+    - cron: 0 0 1 * *
+
+jobs:
+  build-e2e-docker-image:
+    runs-on: ubuntu-latest
+    name: Publish E2E Frontend Image
+
+    steps:
+      - name: Cloning repo
+        uses: actions/checkout@v3
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{github.actor}}
+          password: ${{secrets.GITHUB_TOKEN}}
+
+      - name: Build E2E Frontend Image
+        run: |
+          cd frontend
+          docker build -f Dockerfile-base.e2e --tag ghcr.io/flagsmith/e2e-frontend:latest .
```

Comment:
Should we rename the tag to something like `ghcr.io/flagsmith/e2e-frontend-base:latest` or something similar so that we don't get confused between the base image and the actual image? Thoughts?

Reply:
OK yep changed